pithos/plugins/_mpris.py

```diff
@@ -576,6 +581,17 @@ def CanEditTracks(self):
         '''b Read only Interface MediaPlayer2.TrackList'''
         return False
 
+    @dbus_property(MEDIA_PLAYER2_RATINGS_IFACE, signature='b')
+    def CanRate(self):
```

Comment:
This seems very specific to Pithos so maybe this should have a more specific name?